test/python/circuit/test_gate_power.py

```diff
@@ -30,7 +30,7 @@ class TestPowerSgate(QiskitTestCase):
     @data(2, 1, 0, -1, -2)
     def test_sgate_int(self, n):
         """Test Sgate.power(n) method with n as integer."""
-        result = SGate().power(n)
+        result = SGate() ** n
```

Comment:
The `Gate.power` method is still going to be a defined part of the interface, so it'd be better to leave its direct tests untouched.  Really, I think the behaviour we're trying to test is just that `Gate.__pow__` works and calls `Gate.power` - logically, I think we only need one single additional test that explicitly does that.

Reply:
Done.